wgpu/Cargo.toml

```diff
@@ -197,19 +148,47 @@ serde = { workspace = true, features = ["default", "derive"], optional = true }
 smallvec.workspace = true
 static_assertions.workspace = true
 
-[dependencies.naga]
-workspace = true
-optional = true
+[target.'cfg(any(target_arch = "wasm32", not(target_os = "emscripten")))'.dependencies]
+js-sys = { workspace = true, features = ["default"] }
 
-[build-dependencies]
-cfg_aliases.workspace = true
+########################################
+# Target Specific Feature Dependencies #
+########################################
```

Comment:
This section was recast such that instead of trying to get exact right combination of targets for each feature, we just list out the target categories and list out each feature. This is soooo much more clear.